pkg/datadog/zaplogger.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package datadog // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/datadog"
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+)
+
+// Zaplogger implements the tracelog.Logger interface on top of a zap.Logger
```

Comment:
Should we add a compile time interface assertion?

```
var _ tracelog.Logger = &Zaplogger{}
```

Reply:
Good point, added the assertion